frontend/src/mixins/shootItem.js

```diff
@@ -132,6 +132,9 @@ export const shootItem = {
     shootDomain () {
       return get(this.shootSpec, 'dns.domain')
     },
+    shootDnsProviders () {
```

Comment:
I think this should not be part of this PR. Correct me if I am wrong.

Reply:
correct, I removed it